### DISCUSSION TITLE
PatchTool 6.21 10/12

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#234] [Announcements]
翻譯更新至中國服6.15，修正翻譯，刪減檔案大小。

檔案：
- FFXIVChnTextPatch-20221012.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：
- rawexd-zhcn-20221012.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- backup-20221012.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

目前測試，在之前的更新後，第三方4k字體包可能會導致遊戲崩潰。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.70'>PatchTool 6.21 10/12</a>.</em>

**Comment by okami2223** [comment]
作者不好意思，請教下
未來是否會有對應的4K字體呢，覺得以前的字體清晰感比較舒服
或者有什麼替代方案能夠取代三方4K字體呢，再麻煩作者或其他大大們解惑，謝謝。
